contracts/DCAHubSwapper/DCAHubSwapper.sol

```diff
@@ -1,15 +1,8 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 pragma solidity >=0.8.7 <0.9.0;
 
-import './DCAHubSwapperDustHandler.sol';
-import './DCAHubSwapperParameters.sol';
 import './DCAHubSwapperSwapHandler.sol';
 
-contract DCAHubSwapper is DCAHubSwapperParameters, DCAHubSwapperSwapHandler, DCAHubSwapperDustHandler, IDCAHubSwapper {
-  constructor(
-    IDCAHub _hub,
-    IWrappedProtocolToken _wToken,
-    address _governor,
-    address _swapperRegistry
-  ) DCAHubSwapperParameters(_hub, _wToken, _governor) DCAHubSwapperSwapHandler(_swapperRegistry) {}
+contract DCAHubSwapper is DCAHubSwapperSwapHandler, IDCAHubSwapper {
```

Comment:
If this is going to inherit only this, shouldn't it be the whole contract?

Reply:
We still need to add a few things. I'm not sure if it will be added to this handler, or to another. If this ends up being the only handler, we will make it the "whole" contract